src/views/ManageList.tsx

```diff
@@ -76,6 +85,43 @@ export function ManageList({ listPath }: Props) {
 		}
 	};
 
+	const handleInvite = async (
+		e: FormEvent<HTMLFormElement>,
+		listPath: string,
+	) => {
+		console.log("Button clicked! Inviting user!");
+		e.preventDefault();
+
+		const trimmedEmailName = validateTrimmedString(emailName);
```

Comment:
praise: I love seeing this util function `validateTrimmedString` being used again! How cool!

Reply:
I do too! Except I don't think it needs to be 😄 because the input element has the `email` type, it isn't possible to get this component into a state where `email` has any whitespace, the input trims it for you already:

https://html.spec.whatwg.org/multipage/input.html#email-state-(type=email)

Reply:
and for what it's worth, I didn't know this either 😄 I knew `email` wouldn't let [an empty string through](https://github.com/the-collab-lab/tcl-77-smart-shopping-list/pull/28/files#r1739852790) (the benefit of using `email` is that you don't have to worry about figuring out if the string is an email, which can't be an empty string). But I discovered through testing that it was _also_ stripping the white space, which is very good to know!